modules/cockroachdb/cockroachdb.go

```diff
@@ -0,0 +1,111 @@
+package cockroachdb
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/url"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	defaultSQLPort   = "26257"
+	defaultAdminPort = "8080"
+
+	defaultImage     = "cockroachdb/cockroach"
+	defaultImageTag  = "latest-v23.1"
+	defaultDatabase  = "defaultdb"
+	defaultStoreSize = "100%"
+)
+
+// CockroachDBContainer represents the CockroachDB container type used in the module
+type CockroachDBContainer struct {
+	testcontainers.Container
+	opts Options
+}
+
+// MustConnectionString panics if the address cannot be determined.
+func (c *CockroachDBContainer) MustConnectionString(ctx context.Context) string {
+	addr, err := c.ConnectionString(ctx)
+	if err != nil {
+		panic(err)
+	}
+	return addr
+}
```

Comment:
@mdelapenya is this ok?

Reply:
Thanks! Also, asking because I don't see this for other database modules

Reply:
No, it's a patter that we should add as a consequence, as I like having both methods